std_msgs/String object
sensor_msgs/Image frame
---
geometry_msgs/Point[] coordinates